Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM node:16.13.0-alpine as build
+
+WORKDIR /app
+
+COPY package.json .yarnrc.yml /app/
+COPY .yarn /app/.yarn
+COPY packages/@uppy/companion /app/packages/@uppy/companion
+
+RUN apk --update add  --virtual native-dep \
+  make gcc g++ python3 libgcc libstdc++ git && \
+  cd /app && corepack yarn plugin import workspace-tools && \
```

Comment:
The `workspace-tools` should be already in `/app/.yarn`, can we remove this instruction?

Reply:
oops i missed that,thanks!